doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,62 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: Since the primary purpose of Dataverse is to serve as an archive and to track data accesses and the modifications of data and metadata, a simple mechanism to delete users that have performed certain actions in the system is not provided by design. 
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
+
+Disabling a user with this endpoint will keep:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author.
+- The user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use.
```

Comment:
We don't track acceptance of terms of use (or if we do, I could use a pointer).

I'm confused about why we're saying that the user's access to datafiles is preserved. Aren't we revoking all permissions when the user is disabled? If not, shouldn't this be broader, about datasets and dataverses too?

Reply:
removed submit for review/returning to author in https://github.com/IQSS/dataverse/pull/7585/commits/eafdc73ecde663d04f6790b19bb5be57b8883932

added "history" after access to make it more clear in https://github.com/IQSS/dataverse/pull/7585/commits/eafdc73ecde663d04f6790b19bb5be57b8883932